Source/WebCore/platform/cocoa/RuntimeApplicationChecksCocoa.mm

```diff
@@ -158,12 +152,6 @@ static bool applicationBundleIsEqualTo(const String& bundleIdentifierString)
     return isAdobeInstaller;
 }
 
-bool MacApplication::isAOLInstantMessenger()
-{
-    static bool isAOLInstantMessenger = applicationBundleIsEqualTo("com.aol.aim.desktop"_s);
```

Comment:
This application has been decommissioned, and any modern variant is unlikely to need this quirk.